docs/source/system-metrics/index.rst

```diff
@@ -0,0 +1,214 @@
+
+.. code-block:: bash
+
+    export MLFLOW_ENABLE_SYSTEM_METRICS_LOGGING="false"
+
+.. code-block:: python
+
+    import os
+
+    del os.environ["MLFLOW_ENABLE_SYSTEM_METRICS_LOGGING"]
+
+Rerunning the MLflow code above will not log system metrics.
+
+Use ``mlflow.enable_system_metrics_logging()``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We also provide a pair of APIs ``mlflow.enable_system_metrics_logging()`` and
+``mlflow.disable_system_metrics_logging()`` to turn on/off system metrics logging globally, since
+sometimes you won't be able to set the environment variable. Running the following code will have the
+same effect as setting ``MLFLOW_ENABLE_SYSTEM_METRICS_LOGGING`` environment variable to ``true``:
+
+.. code-block:: python
+
+    import mlflow
+
+    mlflow.enable_system_metrics_logging()
+
+    with mlflow.start_run() as run:
+        time.sleep(15)
+
+    print(mlflow.MlflowClient().get_run(run.info.run_id).data)
+
+Turn on System Metrics Logging for a Single Run
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In addition to controlling system metrics logging globally, you can also control it for a
+single run. To do so, set ``log_system_metrics`` as ``True`` or ``False`` accordingly in :py:func:`mlflow.start_run()`:
+
+.. code-block:: python
+
+    with mlflow.start_run(log_system_metrics=True) as run:
+        time.sleep(15)
+
+    print(mlflow.MlflowClient().get_run(run.info.run_id).data)
+
+The above code will log system metrics for the current run, even if you have disabled system metrics logging by setting
+``MLFLOW_ENABLE_SYSTEM_METRICS_LOGGING`` to ``false`` or calling ``mlflow.disable_system_metrics_logging()``.
+
+Type of System Metrics
+----------------------
+
+By default MLflow logs the following system metrics:
+
+* cpu_utilization_percentage
+* system_memory_usage_megabytes
+* system_memory_usage_percentage
+* gpu_utilization_percentage
+* gpu_memory_usage_megabytes
+* gpu_memory_usage_percentage
+* network_receive_megabytes
+* network_transmit_megabytes
+* disk_usage_megabytes
+* disk_available_megabytes
+
+GPU metrics are only logged when GPU is available and ``pynvml`` is installed.
+
+Every system metric has a prefix ``system/`` when logged for grouping purpose. So the actual metric name gets logged
+are ``system/cpu_utilization_percentage``, ``system/system_memory_usage_megabytes``, etc.
+
+View System Metrics on MLflow UI
+--------------------------------
+
+System metrics are available on MLflow UI under the metrics section, let's start our MLflow UI server, and log system
+metrics to it:
+
+.. code-block:: bash
+
+    mlflow ui
+
+.. code-block:: python
+
+    import mlflow
+    import time
+
+    mlflow.set_tracking_uri("http://127.0.0.1:5000")
+    with mlflow.start_run() as run:
+        time.sleep(15)
+
+Navigate to `http://127.0.0.1:5000 <https://docs.databricks.com/en/machine-learning/index.html>`_ in your browser,
```

Comment:
This shouldn't be a hyperlink, right? Just an inline code block?

Reply:
yes, maybe a fat finger there..